server/autolinkplugin/command.go

```diff
@@ -85,7 +88,15 @@ func (p *Plugin) ExecuteCommand(c *plugin.Context, commandArgs *model.CommandArg
 }
 
 func executeList(p *Plugin, c *plugin.Context, header *model.CommandArgs, args ...string) *model.CommandResponse {
-	links, refs, err := parseLinkRef(p, false, args...)
+	var links []autolink.Autolink
+	var refs []int
+	var err error
+
+	if len(args) > 0 && (args[0] == template || args[0] == pattern) {
+		links, refs, err = parseLinkRefByTemplateOrPattern(p, header, args...)
```

Comment:
0/5 rename the `parseLink...` functions to `searchLink...` to be more intention-revealing?